### ISSUE TITLE
Reading messages while sync is in progress marks them as unread again after sync is complete

### ISSUE THREAD

**Issue by Bubu** [thunderbird/thunderbird-android#5554]
**Describe the bug**
When a message sync is currently in progress and you read some messages in the meantime, they are marked as unread again after the sync finishes.

**To Reproduce**
1. Open K-9
2. Pull down to refresh
3. Open an unread message
4. Go back
5. wait for sync to complete
6. See message as unread again

**Expected behavior**
Messages stay read/unread even if a concurrent sync is running

**Environment (please complete the following information):**
 - K-9 Mail version: 5.802
 - Android version: 8.1
 - Device: Nexus 5X
 - Account type: Imap

**Comment by erwin68** [comment]
For me, all messages received since August 9 remain unread, no matter how many times I open the message in K9. After the next sync, the messages appear again as unread.

If I look at the mail account in parallel from the desktop with Thunderbird, then I see that opening the message in K9 is not synchronized to the mail server.

The problem is very annoying.

**Environment**

- K-9 Mail version: 5.802
- Android version: 10
- Device: Samsung S9Plus
- Account type: Imap (1und1.de)

**Comment by Bubu** [comment]
@erwin68 you should open a separate issue, as this sounds like an unrelated problem. Syncing the read status *does* work for me in general.
